internal/operator/operator.go

```diff
@@ -59,6 +59,7 @@ type Operator struct {
 // +kubebuilder:rbac:groups=operator.projectcontour.io,resources=contours/status,verbs=get;update;patch
 // cert-gen needs create/update secrets.
 // +kubebuilder:rbac:groups="",resources=namespaces;secrets;serviceaccounts;services,verbs=get;list;watch;delete;create;update
+// +kubebuilder:rbac:groups="",resources=secrets;serviceaccounts;services,verbs=get;list;watch;delete;create;update
```

Comment:
these two lines are identical except the first also includes `namespaces`, should be de-duped.